test/OpenTelemetry.Collector.AspNet.Tests/OpenTelemetry.Collector.AspNet.Tests.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Unit test project for OpenTelemetry</Description>
```

Comment:
minor: change description

Reply:
A bunch of the test projects had descriptions that didn't match. I went through and updated them. Will be on my next commit.